src/pages/page.tsx

```diff
@@ -0,0 +1,124 @@
+import { Container } from '@chakra-ui/react';
+import { useState, useEffect } from 'react';
+
+type DriveFile = {
+  id: string;
+  name: string;
+};
+
+export default function FileUploadForm() {
+  const [file, setFile] = useState(null);
+  const [fileName, setFileName] = useState('');
+  const [driveFiles, setDriveFiles] = useState<DriveFile[]>([]);
+
+  const handleFileChange = (e) => {
+    setFile(e.target.files[0]);
+    setFileName(e.target.files[0].name);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (!file) return;
+
+    const formData = new FormData();
+    formData.append('file', file);
+    formData.append('fileName', fileName);
+
+    try {
+      const res = await fetch('/api/uploadFile/', {
+        method: 'POST',
+        body: formData,
+      });
+
+      const data = await res.json();
+      if (res.ok) {
+        const fakeEvent = { preventDefault: () => {} };
```

Comment:
Using `fakeEvent` is unnecessary. You can directly call `handleSearch` without arguments if you refactor it not to rely on the event object.